.github/workflows/publish-package.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   deploy:
     if: github.event.pull_request.merged == true && startsWith(github.head_ref, 'release/')
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
this should fix the issue with the last release not being able to tag the repo with the new version